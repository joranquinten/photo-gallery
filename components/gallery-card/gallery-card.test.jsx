import React from 'react';
import { shallow } from 'enzyme';

import GalleryCard from './gallery-card';

describe('GalleryCard', () => {
  it('should render without crashing', () => {
    expect(shallow(<GalleryCard title="title" slug="slug" />)).toMatchSnapshot();
  });

  it('should render with description as HTML', () => {
    expect(
      shallow(<GalleryCard title="title" slug="slug" description="<p>Description as HTML</p>" />)
    ).toMatchSnapshot();
  });
});
