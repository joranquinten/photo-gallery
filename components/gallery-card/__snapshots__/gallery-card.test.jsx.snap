// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GalleryCard should render with description as HTML 1`] = `
<li>
  <h2>
    <Link
      as="/slug"
      href="/?slug=slug"
      prefetch={true}
    >
      <a>
        title
      </a>
    </Link>
  </h2>
  <p
    key="0"
  >
    Description as HTML
  </p>
</li>
`;

exports[`GalleryCard should render without crashing 1`] = `
<li>
  <h2>
    <Link
      as="/slug"
      href="/?slug=slug"
      prefetch={true}
    >
      <a>
        title
      </a>
    </Link>
  </h2>
</li>
`;
